message_definitions/v1.0/messages.xml

```diff
@@ -266,12 +260,7 @@
     <field name="value" type="float"/>
   </message>
 
-
-  <message name="MARK" id="32">
```

Comment:
This one is used by the GCS to leave a mark on the map (the button 'mark' on the strip that probably nobody knows...)